### PR TITLE
Fixing Call bridge field parsing (API returns the complete bridge URL…

### PR DESCRIPTION
**Pull request by lvivo-daitan**
Fixing the call bridge_id field (API returns the complete bridge URL instead of the bridge id itself)
Also updating SDK tests in order to reflect the fix

### CODE REVIEW

**Comment by lfavaro-daitan** [comment]
LGTM

File: bandwidth_sdk/models.py [hunk 1]
```diff
@@ -132,6 +132,9 @@ def set_up(self, data):
         self.from_ = self.from_ or data.get('from')
         self.call_id = self.call_id or data.get('id')
         super(Call, self).set_up(data)
+        bridge = data.get('bridge')
+        if bridge is not None:
+            self.bridge_id = bridge_parts = bridge.split("/")[-1]
```
Comment (lnunes-daitan) [inline bandwidth_sdk/models.py]: I think there is a mistake in here

File: bandwidth_sdk/models.py [hunk 2]
```diff
@@ -132,6 +132,9 @@ def set_up(self, data):
         self.from_ = self.from_ or data.get('from')
         self.call_id = self.call_id or data.get('id')
         super(Call, self).set_up(data)
+        bridge = data.get('bridge')
+        if bridge is not None:
+            self.bridge_id = bridge.split("/")[-1]
```
Comment (dpereira-daitan) [inline bandwidth_sdk/models.py]: use ' instead of "

File: bandwidth_sdk/models.py (hunk 1 above)
Comment (lvivo-daitan) [inline bandwidth_sdk/models.py]: Removed the unused variable :innocent:

**Comment by dtolb** [comment]
@mzizzi @lfavaro-daitan @dpereira-daitan how do we update this on PIP ?

**Comment by dtolb** [comment]
Released v1.10 https://pypi.python.org/pypi/bandwidth-sdk

**Pull request merged**